clang/docs/ClangFormatStyleOptions.rst

```diff
@@ -6237,18 +6237,30 @@ the configuration (without a prefix: ``Auto``).
        true:                                  false:
        x = ( int32 )y                 vs.     x = (int32)y
 
-  * ``bool InEmptyParentheses`` Put a space in parentheses only if the parentheses are empty i.e. '()'
+  * ``bool InEmptyParentheses`` Put a space in parentheses and braces only if they are empty i.e. '()' or '{}'
```

Comment:
Did you make this change by hand or by using the script in  docs/tools

Reply:
I made by hand. I didn't notice the script, (and still don't know what to be used...)

Reply:
The script is located at `clang/docs/tools/dump_format_style.py`.